suite-native/module-send/src/sendOutputsFormSchema.ts

```diff
@@ -88,12 +96,17 @@ export const sendOutputsFormValidationSchema = yup.object({
                         'is-invalid-address',
                         'The address format is incorrect.',
                         (value, { options: { context } }: yup.TestContext<SendFormFormContext>) => {
-                            const networkSymbol = context?.networkSymbol;
+                            if (!value || !context) {
+                                return false;
+                            }
+                            const { networkSymbol, isTaprootUnavailable } = context;
 
                             return (
-                                G.isNotNullable(value) &&
                                 G.isNotNullable(networkSymbol) &&
-                                isAddressValid(value, networkSymbol)
+                                isAddressValid(value, networkSymbol) &&
+                                !isAddressDeprecated(value, networkSymbol) &&
+                                !isBech32AddressUppercase(value) && // bech32 addresses are valid as uppercase but are not accepted by Trezor
+                                !(isTaprootAddress(value, networkSymbol) && isTaprootUnavailable) // bech32m/Taproot addresses are valid but may not be supported by older FW
```

Comment:
I'm not a fan of `!` in front of a complex boolean expression. I'd rename `isTaprootUnavailable` to `isTaprootAvailable` and simplify the negation.

Reply:
fixup 9c98eccf9d